test/vm/scripts/compile_scap_open.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -eou pipefail
+
+echo "Building scap-open"
+
+LIBS_DIR="/libs"; # dir mounted in container
+mkdir -p "${LIBS_DIR}/build";
+LIBS_TAR_GZ="/vm/build/libs-src.tar.gz"; # host dir mounted in container
+
+if [[ ! -f ${LIBS_TAR_GZ} ]]; then
+  printf "\n\n[STATUS] FAILED Container requires libs source in libs-src.tar.gz format and mounted to /vm/build/libs-src.tar.gz in container\n\n"
+  exit 1
+fi
+
+tar -xvf ${LIBS_TAR_GZ} -C ${LIBS_DIR}/;  # fresh extraction of libs src in container, clean build dir
```

Comment:
Same as a previous comment, don't you need to `rm -rf "${LIBS_DIR}"` to ensure the directory is clean?